tests/installation/bootloader_uefi.pm

```diff
@@ -34,6 +34,12 @@ sub run() {
         send_key "ret";
     }
 
+    # Skip to load bootloader in test of online migration on aarch64
+    # Handle aarch64 image boot by wait_boot called in setup_online_migration
+    if (get_var('ONLINE_MIGRATION') && check_var('ARCH', 'aarch64')) {
+        return;
+    }
```

Comment:
I only have x86_64 machines available to test on my site.
And I understand what you suggested which I have also concerned. Offline migration is using boot_to_desktop while online migration is not according to early design, so the next step I would like to reconstruct and unify the booting and common used module for both of them on variety archs. I will start it after SP3 phrase.